service/src/main/java/bio/terra/landingzone/service/landingzone/azure/LandingZoneService.java

```diff
@@ -119,23 +193,32 @@ public List<LandingZoneResource> listResourcesByPurpose(
         .collect(Collectors.toList());
   }
 
-  public List<String> listLandingZoneIds(LandingZoneTarget landingZoneTarget) {
```

Comment:
This doesn't appear to be used from WSM so I just removed it.

We may want a "list landing zones" endpoint endpoint eventually, but need to decide the right semantics/use case I think:
- Get landing zone by id
- List landing zones in a billing profile
- List all landing zones the caller has access to
?

Reply:
I use this method to look for a LZ id from the WSM Azure context in the VM create operation. We will need to refactor/remove this method as a follow-up, and once we define the proper flow within WSM for getting the LZ id. But right now, we can't remove this method.

Reply:
OK good to know! I'll add it back (and rebase my WSM PR with the VM creation changes).

Reply:
@giventocode to follow up, I added back the methods to list LZs by `LandingZoneTarget` and by `billingProfileId`. I think this serves WSM's needs for now, since it can continue to query for a LZ based on the existing `CloudContext`. 

We can discuss later the desired changes to the cloud context, if any.